### PR TITLE
fmt(cstyle): fix cstyle issue in src/mock_client.c

### PR DESCRIPTION
**Pull request by oppiliappan**

### CODE REVIEW

File: src/mock_client.c [hunk 1]
```diff
@@ -31,21 +31,21 @@ void *mgmt_thrd(void *args);
...
-static void build_cmd(cargs_t *cargs, ISTGT_LU_CMD_Ptr lu_cmd, SBC_OPCODE opcode,
-    int len);
+static void build_cmd(cargs_t *cargs, ISTGT_LU_CMD_Ptr lu_cmd,
+    SBC_OPCODE opcode, int len);
 
 cargs_t *all_cargs = NULL;
 pthread_t *all_cthreads = NULL;
 
-static void
+	static void
```
Comment (utkarshmani1997) [inline src/mock_client.c]: Fix indentation

File: src/mock_client.c [hunk 2]
```diff
@@ -74,19 +74,18 @@ build_cmd(cargs_t *cargs, ISTGT_LU_CMD_Ptr cmd, SBC_OPCODE opcode,
...
-	return;
 }
 
 /*
  * Adds write IOs to replication module
  * Increments '*count' variable to set the completion of writer thread
  */
-void *
+	void *
 writer(void *args)
```
Comment (utkarshmani1997) [inline src/mock_client.c]: fix the same here

File: src/mock_client.c [hunk 3]
```diff
@@ -158,14 +157,14 @@ writer(void *args)
...
+	return (NULL);
 }
 
 /*
  * Adds read IOs to replication module
  * Increments '*count' variable to set the completion of reader thread
  */
-void *
+	void *
 reader(void *args)
```
Comment (utkarshmani1997) [inline src/mock_client.c]: same here

File: src/mock_client.c [hunk 4]
```diff
@@ -237,10 +236,10 @@ reader(void *args)
 	pthread_cond_signal(cv);
 	MTX_UNLOCK(mtx);
 
-	return NULL;
+	return (NULL);
 }
 
-void *
+	void *
 mgmt_thrd(void *args)
```
Comment (utkarshmani1997) [inline src/mock_client.c]: same here

File: src/mock_client.c [hunk 5]
```diff
@@ -287,15 +287,16 @@ mgmt_thrd(void *args)
...
- * creates client threads that are needed to send read/write IOs to replication module.
- * cargs_t stores details that are sent to reader/writer threads
- * which isends IOs to replication module.
+ * creates client threads that are needed to send read/write IOs to replication
+ * module.  cargs_t stores details that are sent to reader/writer threads which
+ * isends IOs to replication module.
  */
-void
+
+	void
```
Comment (utkarshmani1997) [inline src/mock_client.c]: same here

File: src/mock_client.c [hunk 6]
```diff
@@ -350,19 +360,17 @@ create_mock_client(spec_t *spec)
 	free(all_cthreads);
 	all_cthreads = NULL;
 	all_cargs = NULL;
-
-	return;
 }
 
-void
+	void
```
Comment (utkarshmani1997) [inline src/mock_client.c]: same here

**Review by utkarshmani1997 — COMMENTED** [review]
Changes are good. Please fix the indentation and update the description with the output of `./cstyle.pl filename`

**Review by vishnuitta — APPROVED** [review]
changes are good

**Pull request merged**